satpy/resample.py

```diff
@@ -109,6 +109,14 @@
     >>> my_area = AreaDefinition(...)
     >>> local_scene = scn.resample(my_area)
 
+Resize area definition in pixels
+--------------------------------
+
+Sometimes you may want to create a small image with fixed size in pixels.
+For example, to create an image of (x, y) pixels :
+
+    >>> small_scn = scn.resample(scn.finest_area().copy(height=y, width=x), ...)
```

Comment:
It may be worth mentioning that this won't always work with native resampling if it isn't a factor of the original and/or show `resampler="nearest"` in the code just to suggest that the resampler should be specified.

Reply:
Yes i'm adding this in the example to be clearer